public/static/docs/command-reference/checkout.md

```diff
@@ -143,9 +143,9 @@ bigrams-experiment      <- use bigrams to improve the model
 ```
 
 This project comes with a predefined HTTP
-[remote storage](/doc/command-reference/remote). We can now just run `dvc pull`
-that will fetch and checkout the most recent `model.pkl`, `data.xml`, and other
-files that are under DVC control. The model file checksum
+[remote storage](/doc/command-reference/remote#description). We can now just run
```

Comment:
if we do basic concepts, let's avoid changes like this - we already have 35 files changed. It will quickly become an extremely hard to review thing.